<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nombreUsuarioLabel">
        <property name="text">
         <string>Nombre Del Usuario</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nombreUsuarioLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="correosLabel">
        <property name="text">
         <string>Correos de Emergencia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="correosTextEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="guardarCorreosButton">
        <property name="text">
         <string>Guardar Correos</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="buscarHorarioLabel">
          <property name="text">
           <string>Buscar d√≠as por hora</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="buscarHorarioInput">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Ingrese una hora (hh:mm)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buscarButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resultadosLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="scheduleTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>7</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>57</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>176</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <item row="4" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guardarHorariosButton">
        <property name="text">
         <string>Guardar Horarios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultadosLabel">
        <property name="text">
         <string>Resultados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buscarExtremosButton">
        <property name="text">
         <string>Buscar Horarios Extremos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
